tests/jballerina-unit-test/src/test/resources/test-src/query/query-expr-with-query-construct-type.bal

```diff
@@ -624,7 +621,6 @@ type TokenTable table<Token> key(idx);
 
 function testQueryConstructingTableWithOnConflictClauseHavingNonTableQueryInLetClause() {
     TokenTable|error tbl1 = table key(idx) from int i in 1 ... 3
-        let int[] arr = from var j in 1 ... 3 select j
```

Comment:
Why are we removing this?

Reply:
There was an unused variable warning. Then we can improve this test with use this arr variable. Will revert the changes.

Reply:
My bad. I didn't notice the test title (... InLetCluase). Shall I update this test with using `arr` in the select clause to avoid unused variable warning. WDYT @lasinicl ?

Reply:
that would be okay.

Reply:
I still see the deletion in the diff

Reply:
will check

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/36743/commits/64c0b2d78125625a7b66f5a8070b505115c99de0.